### PR TITLE
Update default version value in documentation

### PR DESCRIPTION
**Pull request by GintV**

### CODE REVIEW

File: common/changes/@itwin/imodels-client-management/fix-documentation_2023-08-24-09-44.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
```
Comment (austeja-bentley) [inline common/changes/@itwin/imodels-client-management/fix-documentation_2023-08-24-09-44.json:2]: We do not have changelog files, please remove.

**Review by austeja-bentley — CHANGES_REQUESTED** [review]

File: common/changes/@itwin/imodels-client-management/fix-documentation_2023-08-24-09-44.json (hunk 1 above)
Comment (GintV) [reply common/changes/@itwin/imodels-client-management/fix-documentation_2023-08-24-09-44.json]: Done

**Review by austeja-bentley — APPROVED** [review]
thank you for the fix!

**Pull request merged**